packages/manager/apps/cloud/client/app/dbaas/logs/detail/logs-detail.less

```diff
@@ -15,6 +15,19 @@
   }
 }
 
+ul.overview {
+  list-style-type: none;
+}
+
+ul.overview > li {
+  text-indent: -5px;
```

Comment:
we prefer to use rem sizing if possible

Reply:
No more a subject since additional style has been removed from this PR.